mii/client.py

```diff
@@ -34,6 +34,10 @@ def mii_query_handle(deployment_name):
     Returns:
         query_handle: A query handle with a single method `.query(request_dictionary)` using which queries can be sent to the model.
     """
+    if deployment_name in mii.non_persistent_models:
+        inference_pipeline, task = mii.non_persistent_models[deployment_name]
```

Comment:
Shouldn't we explicitly specify if we want to use a non-persistent model or not?